scripts/tests/staking_scenario.js

```diff
@@ -1,10 +1,10 @@
-const common = require('./common/common');
-const {range} = require('../test/utils');
+const common = require('../common/common');
```

Comment:
Did you do a `git mv` here? That's what I wanted to see in `deployment_test.js`.

Reply:
no git mv.